mozjs-sys/etc/filters.txt

```diff
@@ -1,3 +1,7 @@
+# we do not need testing infra
+- /js/src/fuzz-tests
+- /js/src/jit-test
```

Comment:
There is actually more testing stuff that we could remove, but this have some rust in them, so they are more problematic and not really needed. There are also jsapi-test that are kept for now.